spec/helpers/articles_helper_spec.rb

```diff
@@ -22,4 +22,35 @@
       expect(host).to eq "example.com"
     end
   end
+
+  describe "#image_tag_or_inline_svg" do
+    helper do
+      def internal_navigation?
+        true
+      end
+    end
+    subject { helper.image_tag_or_inline_svg("twitter") }
+
+    it { is_expected.to start_with("<img") }
+
+    context "with a width and height" do
+      subject { helper.image_tag_or_inline_svg("twitter", width: 18, height: 18) }
+
+      it { is_expected.to include('width="18" height="18"') }
```

Comment:
I'm only checked if the attributes exist because the order the attributes would output was different on my local machine vs CI.